packages/components/socialFeed/NewsFeed/NewsFeedInput.tsx

```diff
@@ -141,7 +141,8 @@ export const NewsFeedInput = React.forwardRef<
     const selectedNetwork = useSelectedNetworkInfo();
     const selectedNetworkId = selectedNetwork?.id || "teritori";
     const selectedWallet = useSelectedWallet();
-    const userId = getUserId(selectedNetworkId, selectedWallet?.address);
+    const posterId = daoId ? daoId : selectedWallet?.userId;
```

Comment:
nit, this can be simplified as

```suggestion
    const posterId = daoId || selectedWallet?.userId;
```